sapientml_core/templates/other_templates/evaluation.py.jinja

```diff
@@ -16,14 +16,22 @@ from sklearn.metrics import accuracy_score
 
 __accs = []
 for i, col in enumerate(target_test.columns):
-    one_acc = accuracy_score(target_test[col], y_pred[:, i:i+1])
+    one_acc = accuracy_score(target_test[col], y_pred[:, col])
```

Comment:
`y_pred[:, col]` -> `y_pred[col]`

Reply:
Nice catch. Fix in [85e3c77](https://github.com/sapientml/core/pull/59/commits/85e3c77ca95864e4307152fb83865c2992811c85)